bootloader/assets/grub.go

```diff
@@ -32,16 +32,16 @@ package assets
 //
 //     gadget: bump edition to 2, using production signing keys for everything.
 
-const grubBootConfig = `# Snapd-Boot-Config-Edition: 1
+const grubBootConfig = `# Snapd-Boot-Config-Edition: 2
```

Comment:
why the edition bump? the change here is purely cosmetic no?

Reply:
Force pushed instead of reverting the last patch.